themes/bootstrap3/templates/RecordDriver/EDS/core.phtml

```diff
@@ -23,36 +23,37 @@
           </a>
         </span>
       <?php endif; ?>
-      <?php $ftLink = $this->driver->getLinkedFullTextLink();
-          if ($ftLink): ?>
-        <span>
-          <a href="<?=$ftLink?>" class="fulltext">
-            <?=$this->transEsc('Linked Full Text')?>
-          </a>
-        </span>
-      <?php endif; ?>
-      <?php $pdfLink = $this->driver->getPdfLink();
-          if ($pdfLink): ?>
-        <span>
-          <a href="<?=$pdfLink?>" class="icon--eds pdf fulltext">
-            <?=$this->transEsc('PDF Full Text')?>
-          </a>
-        </span>
-      <?php endif; ?>
-      <?php $epubLink = $this->driver->getEpubLink();
-          if ($epubLink): ?>
-        <span>
-          <a href="<?=$epubLink?>" class="icon--eds epub fulltext">
-            <?=$this->transEsc('ePub Full Text')?>
-          </a>
-        </span>
-      <?php endif; ?>
-      <?php if ($this->driver->hasHTMLFullTextAvailable()): ?>
-        <span>
-          <a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>#html" class="icon--eds html fulltext">
-            <?=$this->transEsc('HTML Full Text')?>
-          </a>
-        </span>
+      <?php if ($this->permission()->isAuthorized('access.EDSExtendedResults')): ?>
+        <?php if ($this->driver->hasHTMLFullTextAvailable()): ?>
+          <span>
+            <a href="<?=$this->escapeHtmlAttr($this->recordLinker()->getUrl($this->driver))?>#html" class="icon--eds html fulltext" target="_blank">
```

Comment:
I think the reason there was no target="_blank" for the HTML full text is that it's an anchor scrolling down to the text on the same page.  I don't love the current display of that full text (i.e. no heading or other visual separation) but that's a separate PR.  But it does seem like it would be odd to open the same page as a separate tab.  Is there a scenario where it would actually be a different URL?

Reply:
Ahh, good point, I didn't catch that. It had a _blank in the search results link, so I just copied that over without thinking too hard to make things consistent. But maybe the _blank should be removed in both contexts, because in both cases, you're staying inside VuFind rather than going off to an EBSCO page. What do you and @sturkel89 think makes the most sense here? Happy to change accordingly, but not totally sure what's most logical.

Reply:
Normally I agree that "are you opening a different website/organization/platform" is the right question for using _blank.  But in this case I think there is logic in the current approach, even though it differs between search-results and record page.  Mostly because I think it's a surprise to people that the full text actually appears within VuFind (it was a surprise to a colleague of mine this morning, and to me a few weeks back), even though on thinking about it I understand why we do it -- it's an API, so why not.  But as I said above I think there is more work to do on the UX how it's embedded in the record page.  

So IMHO I would not actually change the current linking behavior until we work out the rest of the UX separately.

Reply:
When you say "would not actually change the current linking behavior" do you mean "leave the target _blank on HTML links" or "never include target _blank on the record page" or something else? I guess the question is whether you propose maintaining the status quo of the dev branch or of this branch. :-)

Reply:
Sorry, I meant status quo of the dev branch, on that question.  So no UX change for users until we think more about it.

Reply:
The problem is that on the dev branch, all links in search results are target=_blank, and no links on the record page set a target. So by sharing the template here, we introduce an inconsistency. I tried to resolve that by using target=_blank everywhere because that was the simplest fix... but we could also add a new variable to the template to allow us to control whether or not targets are added, and that would allow us to replicate the original behavior. Do you think that's worth the effort?

Reply:
Ok thanks for clarification, I didn't think through those effects of the shared template.  Hmm.  I think yes, I would pass in a variable to control the target attribute, but it would control it *only* for the HTML Full Text link (new tab from the search results, same tab from the record).  And all other links would always be new tab.

Whether that is actually worth the effort though, probably not, and there is consistency in always adding the target attribute, everywhere, even if it's a little odd to do that for same-page results.

But then [WCAG tells us](https://www.w3.org/TR/WCAG20-TECHS/G200.html) basically never to open a new tab for any of these links, regardless where it's hosted.  That's non-intuitive for web devs but [the more I read about it](https://codersblock.com/blog/external-links-new-tabs-and-accessibility/#:~:text=With%20regards%20to%20accessibility%2C%20the,have%20difficulty%20perceiving%20visual%20content.%E2%80%9D), I think it's actually the correct thing to do.  Removing all of the target attributes would also be easy consistency of a different sort.

Whether *any* of that is worth the effort at this point I think is your call.

Reply:
See commit 0d11f9a259ea2d3aa036ddeb4d61c848e4d23496 -- this restores the previous target behavior, but makes using a target configurable via a $newWindow template variable which defaults to false. I'm inclined to agree that using target is generally undesirable, and I would be entirely open to changing the default behavior to never use targets... but since I went to the effort of making a switch, I thought it might make sense to use that switch at least initially, to minimize the change imposed by this PR. We can then flip the switch off separately if we want to make that change.

Reply:
Works for me!